### PR TITLE
[release-0.59]Split 1.26 operator lane to conf. and upgrade

### PR DESCRIPTION
**Pull request by xpivarc**
Backport of https://github.com/kubevirt/project-infra/pull/2587

/hold

Depends on https://github.com/kubevirt/kubevirt/pull/9247

### CODE REVIEW

**Review by brianmcarey — APPROVED** [review]
/lgtm

/approve

waiting for https://github.com/kubevirt/kubevirt/pull/9247 to be merged

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.59.yaml [hunk 1]
```diff
@@ -2279,7 +2279,45 @@ presubmits:
...
+            memory: 29Gi
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - always_run: true
+    branches:
+    - release-0.59
+    cluster: prow-workloads
+    context: pull-kubevirt-e2e-k8s-1.26-sig-operator
```
Comment (brianmcarey) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.59.yaml:2296]: I think you need to update the context here to match the lanes on main

Comment (xpivarc) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.59.yaml]: Updated, both operator lanes

**Review by brianmcarey — APPROVED** [review]
/approve

**Pull request closed without merging**